MIGRATING.md

```diff
@@ -64,32 +79,39 @@ OpenTelemetry Java Instrumentation.
 ### Configuration setting changes
 
 These SignalFx Java Agent system properties correspond to the following
-OpenTelemetry system properties:
+OpenTelemetry system properties (NOTE: some properites are exporter - specific, the default is `jaeger`):
```

Comment:
Typo; remove spaces around the hyphen:

```suggestion
OpenTelemetry system properties (NOTE: some properties are exporter-specific, the default is `jaeger`):
```

> the default is `jaeger`

Default exporter? We use `jaeger-thrift-splunk` by default

Reply:
Yes we do, but properties are generic for all jaeger-based exporters and stating full name here would confuse customers migrating their code.